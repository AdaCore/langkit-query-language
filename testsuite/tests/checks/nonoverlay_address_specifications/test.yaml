driver: 'checker'
rule_name: Nonoverlay_Address_Specifications
input_sources: ['overlay.adb']
