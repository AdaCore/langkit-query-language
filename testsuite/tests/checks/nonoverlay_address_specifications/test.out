overlay.adb:11:24: error: nonoverlay address specification
11 |    Var2 : Integer with Address => Var_Rec.C'Address;  --  FLAG
   |                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 

